frontend/src/metabase/admin/settings/components/ApiKeys/SecretKeyModal.tsx

```diff
@@ -29,7 +29,7 @@ export const SecretKeyModal = ({
         disabled
         styles={{
           input: {
-            color: `black !important`,
+            color: `black`,
```

Comment:
The !important was causing the rule to not be applied (this happened elsewhere as well). Also, this should probably be converted to use CSS modules :/